Sources_v3/Controllers/ChannelController.swift

```diff
@@ -201,6 +233,10 @@ public extension ChannelControllerGeneric {
                        extraData: ExtraData.Channel,
                        completion: ((Error?) -> Void)? = nil)
     {
+        guard isChannelCreated == true else {
+            log.error("You cannot perform channel actions before its creation on backend.")
```

Comment:
suggestion: 
```
You can't modify the channel because the channel hasn't been created yet. Call `startUpdating()` to create the channel and wait for the completion block to finish. Alternatively, you can observe the `state` changes of the controller and wait for the `remoteDataFetched` state.
```